packages/commonwealth/client/scripts/views/components/component_kit/new_designs/cw_search_bar.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { FC, useState } from 'react';
+import { MagnifyingGlass, X } from '@phosphor-icons/react';
+
+import { ComponentType } from './../types';
+
+import 'components/component_kit/new_designs/cw_search_bar.scss';
```

Comment:
as well here `CWSearchBar.scss`

Reply:
Also with the .scss files. Thanks for the tip on using it on the next files/components.